dispatch/dispatch_test.go

```diff
@@ -19,480 +18,304 @@ var (
 
 	// Max out jobs channel
 	for x := 0; x < 99; x++ {
 		err = d.publish(id, "woah-nelly")
-		if !errors.Is(err, nil) {
-			t.Fatalf("received: '%v' but expected: '%v'", err, nil)
-		}
+		assert.NoError(t, err, "publish should not error")
 	}
 
 	err = d.stop()
-	if !errors.Is(err, nil) {
-		t.Fatalf("received: '%v' but expected: '%v'", err, nil)
-	}
-
-	if d.isRunning() {
-		t.Fatalf("received: '%v' but expected: '%v'", d.isRunning(), false)
-	}
+	assert.NoError(t, err, "stop should not error")
+	assert.False(t, d.isRunning(), "IsRunning should return false")
 }
 
 func TestSubscribe(t *testing.T) {
 	t.Parallel()
 	var d *Dispatcher
 	_, err := d.subscribe(uuid.Nil)
-	if !errors.Is(err, errDispatcherNotInitialized) {
-		t.Fatalf("received: '%v' but expected: '%v'", err, errDispatcherNotInitialized)
-	}
+	assert.ErrorIs(t, err, errDispatcherNotInitialized, "subscribe should error correctly")
 
 	d = NewDispatcher()
 
 	_, err = d.subscribe(uuid.Nil)
-	if !errors.Is(err, errIDNotSet) {
-		t.Fatalf("received: '%v' but expected: '%v'", err, errIDNotSet)
-	}
+	assert.ErrorIs(t, err, errIDNotSet, "subscribe should error correctly")
 
 	_, err = d.subscribe(nonEmptyUUID)
-	if !errors.Is(err, ErrNotRunning) {
-		t.Fatalf("received: '%v' but expected: '%v'", err, ErrNotRunning)
-	}
+	assert.ErrorIs(t, err, ErrNotRunning, "subscribe should error correctly")
 
 	err = d.start(0, 0)
-	if !errors.Is(err, nil) {
-		t.Fatalf("received: '%v' but expected: '%v'", err, nil)
-	}
+	require.NoError(t, err, "start should not error")
 
 	id, err := d.getNewID(uuid.NewV4)
-	if !errors.Is(err, nil) {
-		t.Fatalf("received: '%v' but expected: '%v'", err, nil)
-	}
+	require.NoError(t, err, "getNewID should not error")
 
 	_, err = d.subscribe(nonEmptyUUID)
-	if !errors.Is(err, errDispatcherUUIDNotFoundInRouteList) {
-		t.Fatalf("received: '%v' but expected: '%v'", err, errDispatcherUUIDNotFoundInRouteList)
-	}
+	assert.ErrorIs(t, err, errDispatcherUUIDNotFoundInRouteList, "subscribe should error correctly")
 
 	d.outbound.New = func() interface{} { return "omg" }
 	_, err = d.subscribe(id)
-	if !errors.Is(err, errTypeAssertionFailure) {
-		t.Fatalf("received: '%v' but expected: '%v'", err, errTypeAssertionFailure)
-	}
+	assert.ErrorIs(t, err, errTypeAssertionFailure, "subscribe should error correctly")
 
 	d.outbound.New = getChan
 	ch, err := d.subscribe(id)
-	if !errors.Is(err, nil) {
-		t.Fatalf("received: '%v' but expected: '%v'", err, nil)
-	}
-
-	if ch == nil {
-		t.Fatal("expected channel value")
-	}
+	assert.NoError(t, err, "subscribe should not error")
+	assert.NotNil(t, ch, "Channel should not be nil")
 }
 
 func TestUnsubscribe(t *testing.T) {
 	t.Parallel()
 	var d *Dispatcher
 
 	err := d.unsubscribe(uuid.Nil, nil)
-	if !errors.Is(err, errDispatcherNotInitialized) {
-		t.Fatalf("received: '%v' but expected: '%v'", err, errDispatcherNotInitialized)
-	}
+	assert.ErrorIs(t, err, errDispatcherNotInitialized, "X should error correctly")
```

Comment:
```suggestion
	assert.ErrorIs(t, err, errDispatcherNotInitialized, "unsubscribe should error correctly")
```

Reply:
When Regexing the file goes wild.

Fixed in d4aa5a7a _but already rebased back down to_ 4036a1cc since it seemed a slamdunk